src/graph/CMakeLists.txt

```diff
@@ -1,46 +1,47 @@
 add_library(
     graph_obj OBJECT
     GraphFlags.cpp
-    GraphService.cpp
-    ClientSession.cpp
-    SessionManager.cpp
+    service/GraphService.cpp
+    session/ClientSession.cpp
+    session/SessionManager.cpp
     ExecutionEngine.cpp
     ExecutionContext.cpp
     ExecutionPlan.cpp
     Executor.cpp
-    TraverseExecutor.cpp
     SequentialExecutor.cpp
-    UseExecutor.cpp
-    GoExecutor.cpp
-    PipeExecutor.cpp
-    CreateEdgeExecutor.cpp
-    CreateTagExecutor.cpp
-    AlterEdgeExecutor.cpp
-    AlterTagExecutor.cpp
-    DropTagExecutor.cpp
-    DropEdgeExecutor.cpp
-    DescribeTagExecutor.cpp
-    DescribeEdgeExecutor.cpp
-    InsertVertexExecutor.cpp
-    InsertEdgeExecutor.cpp
-    AssignmentExecutor.cpp
-    InterimResult.cpp
-    VariableHolder.cpp
-    AddHostsExecutor.cpp
-    RemoveHostsExecutor.cpp
-    CreateSpaceExecutor.cpp
-    DropSpaceExecutor.cpp
-    DescribeSpaceExecutor.cpp
-    ShowExecutor.cpp
-    YieldExecutor.cpp
-    DownloadExecutor.cpp
-    OrderByExecutor.cpp
-    IngestExecutor.cpp
-    ConfigExecutor.cpp
-    SchemaHelper.cpp
-    FetchVerticesExecutor.cpp
-    FetchEdgesExecutor.cpp
-    FetchExecutor.cpp
+    admin/AddHostsExecutor.cpp
+    admin/RemoveHostsExecutor.cpp
+    admin/CreateSpaceExecutor.cpp
+    admin/DropSpaceExecutor.cpp
+    admin/DescribeSpaceExecutor.cpp
+    admin/ShowExecutor.cpp
+    admin/ConfigExecutor.cpp
+    maintain/CreateEdgeExecutor.cpp
+    maintain/CreateTagExecutor.cpp
+    maintain/AlterEdgeExecutor.cpp
+    maintain/AlterTagExecutor.cpp
+    maintain/DropTagExecutor.cpp
+    maintain/DropEdgeExecutor.cpp
+    maintain/DescribeTagExecutor.cpp
+    maintain/DescribeEdgeExecutor.cpp
+    maintain/SchemaHelper.cpp
+    maintain/YieldExecutor.cpp
+    mutate/InsertVertexExecutor.cpp
+    mutate/InsertEdgeExecutor.cpp
+    mutate/DownloadExecutor.cpp
+    mutate/IngestExecutor.cpp
+	traverse/TraverseExecutor.cpp
```

Comment:
aligmnent

Reply:
I have made modifications according to your comment. And I move ```DownloadExecutor``` to ```query```, move ```IngestExecutor``` to ```update```. 

And what's your opinion about @whitewum's comment?  @sherman-the-tank 

> which is better?
> DDL DML DCL Utility
> or admin tranverse mutate
> We should make our code, manual & ppt consistent with each other

Reply:
I have made modifications according to your comment. And I move ```DownloadExecutor``` to ```query```, move ```IngestExecutor``` to ```update```. 

And what's your opinion about @whitewum's comment?  @sherman-the-tank 

> which is better?
> DDL DML DCL Utility
> or admin tranverse mutate
> We should make our code, manual & ppt consistent with each other